common/src/main/java/com/wynntils/models/containers/BankModel.java

```diff
@@ -53,11 +58,17 @@ public class BankModel extends Model {
     private final Storage<Map<String, Map<Integer, String>>> customCharacterBankPagesNames =
             new Storage<>(new TreeMap<>());
 
+    public static final int LAST_BANK_PAGE_SLOT = 8;
+
+    // Test in BankModel_PERSONAL_STORAGE_PATTERN
+    private static final Pattern PERSONAL_STORAGE_PATTERN =
+            Pattern.compile("^§0\\[Pg\\. (\\d+)\\] §8[a-zA-Z0-9_ ]+'s?§0 (.*)$");
```

Comment:
How come this is not in the enum? It could also return a matcher to fix the problem?

Reply:
This one is just to find current page number for the various banks it's not its own container

Reply:
Okay.